client/webpack-ie11.config.js

```diff
@@ -0,0 +1,29 @@
+const path = require('path');
+module.exports = {
+  entry: {
+    luigiClient: './src/luigi-client.js'
+  },
+
+  output: {
+    filename: 'luigi-client-ie11.js',
+    libraryExport: 'default',
+    library: 'LuigiClientLegacy',
```

Comment:
Maybe we can change the name here as well, to not have `Legacy`

Reply:
Updated to _LuigiClientIe11_